### ISSUE TITLE
Feature request - 2 finger scroll gesture

### ISSUE THREAD

**Issue by sonivraj82** [microsoft/vscode#180218]
There are several gesture which are present in windows 11 and believe me they are of great use. But one of them that dont work great in the vs code app is the two finger scroll gesture. The gesture works but it is too fast, the user is not able to even see the line number. Also the side scroll bar is good and works just fine but it would be great if the gesture function could also be polished and redeveloped correctly and more elegently. The inspiration can be taken from the web browser as the gesture works very fine in the browser and would be great if that same thing could be replicated. Also I will write the same thing in the windows 11 feature request or issue github as I was not sure where it belonged exactly. Also I know this is a very useless feature request ass you guys will have more important issues or feature requests and I'll recommend working on those first, but do look after this if your are free and just wasting microsoft's money as an employee ; )

**Comment by alexdima** [comment]
@sonivraj82 Have you tried configuring `editor.mouseWheelScrollSensitivity`?

**Comment by sonivraj82** [comment]
> @sonivraj82 Have you tried configuring `editor.mouseWheelScrollSensitivity`?

Not yet, maybe need a little help from the internet. Also I would inform if that works.

**Comment by sonivraj82** [comment]
> 

Nope checked it doesn't work. Also if it worked that what should be the value. I tried a lot of values but could not make it smooth, it waas very fast.

**Comment by sonivraj82** [comment]
Hey got the answer on this stackoverflow. It says ->
I know this is an old post, but I was having a similar problem and wanted to share another solution in case anyone else comes across this like I did. The only thing that was sort of working was Ctrl + Touchpad Scroll, but that was really annoying and distracting.

I found another solution on the [Lenovo Support forum](https://forums.lenovo.com/t5/Lenovo-Yoga-Series-Notebooks/Touchpad-not-two-finger-scrolling-properly-in-certain-apps/m-p/4303913?page=1#4304254) that solved my problem in 3 easy steps:

Open regedit.exe using run
Go to Computer\HKEY_CURRENT_USER\Software\Elantech\SmartPad
Change ScrollControl_Mode from 0 to 1
